doc/source/user_guide/computation.rst

```diff
@@ -594,6 +594,32 @@ and we want to use an expanding window where ``use_expanding`` is ``True`` other
    4    10.0
 
 You can view other examples of ``BaseIndexer`` subclasses `here <https://github.com/pandas-dev/pandas/blob/master/pandas/core/window/indexers.py>`__
+One subclass of note within those examples is the ``NonFixedVariableWindowIndexer`` that allows
+rolling operations over a non-fixed offset like a ``BusinessDay``.
+
+.. code-block:: ipython
+
+   In [1]: from pandas.core.window.indexers import NonFixedVariableWindowIndexer
```

Comment:
can you make an ipython block? (e..g execute the code)

Reply:
Wasn't sure if we wanted to advertise importing from core since we want to make that private eventually, but can do nonetheless.